ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/425__model_serving_ovms_on_kserve.robot

```diff
@@ -95,6 +100,49 @@ Verify Multiple Projects With Same Model (Kserve)
     ...    Run Keyword If Test Failed    Get Kserve Events And Logs
     ...    model_name=${MODEL_NAME}    project_title=${PRJ_TITLE}
 
+Verify GPU Model Deployment Via UI (OVMS on Kserve)
+    [Documentation]    Test the deployment of an openvino_ir model on a model server with GPUs attached
+    [Tags]    Sanity    Tier1    Resources-GPU
+    ...       ODS-XXXX
+    Clean All Models Of Current User
+    Open Data Science Projects Home Page
+    Wait For RHODS Dashboard To Load    wait_for_cards=${FALSE}    expected_page=Data Science Projects
+    Create Data Science Project    title=${PRJ_TITLE_GPU}    description=${PRJ_DESCRIPTION_GPU}
+    Create S3 Data Connection    project_title=${PRJ_TITLE_GPU}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=ods-ci-s3
+    Deploy Kserve Model Via UI    model_name=${MODEL_NAME_GPU}    serving_runtime=OpenVINO Model Server
+    ...    data_connection=model-serving-connection    path=vehicle-detection-kserve    model_framework=openvino_ir
+    ...    no_gpus=1
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${MODEL_NAME_GPU}
+    ...    namespace=${PRJ_TITLE_GPU}
+    Verify Displayed GPU Count In Single Model Serving    model_name=${MODEL_NAME_GPU}    no_gpus=1
+    ${requests}=    Get Container Requests    namespace=${PRJ_TITLE_GPU}
+    ...    label=serving.kserve.io/inferenceservice=${MODEL_NAME_GPU}    container_name=kserve-container
+    Should Contain    ${requests}    "nvidia.com/gpu": "1"
+    ${node}=    Get Node Pod Is Running On    namespace=${PRJ_TITLE_GPU}
+    ...    label=serving.kserve.io/inferenceservice=${MODEL_NAME_GPU}
+    ${type}=    Get Instance Type Of Node    ${node}
+    Should Be Equal As Strings    ${type}    "g4dn.xlarge"
```

Comment:
is there a way to check the node has gpu without pinning to the node title? We may run on different GPU types

Reply:
we'll need to make the test compatible with different GPU node types. I think it can be done in another PR once we understand how